### PR TITLE
Class 07

### PR DESCRIPTION
**Pull request by meeksfred**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          <ul id="filters">
+            <li>
+              <select id="category-filter">
+                <option value="">-- Categories --</option>
+              </select>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section id="about" class="tab-content">
```
Comment (meeksfred) [inline index.html:56]: I'm currently working on expanding my 'about' section. I want to add in a 'my favorite/recently found beers' section here, and I'm in the process of getting my initial database setup. This is where I'm going to implement the .reduce() method in conjunction with some other beer statistics using .filter() and .map/forEach() methods. While I'm adding in new content, I'm going to move the about section into a handlebars template as well.

File: scripts/portfolio.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+(function(module) {
+
```
Comment (meeksfred) [inline scripts/portfolio.js:2]: I want to take to heart what Brian was explaining today during class, about adding in my own personal content. This file plus portfolioView.js are fairly optimized at this point, but I'm in the process of brainstorming for new ideas to implement.

File: scripts/portfolio.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  function Project (opts) {
+    this.author = opts.author;
+    this.title = opts.title;
+    this.category = opts.category;
+    this.authorUrl = opts.authorUrl;
+    this.publishedOn = opts.publishedOn;
+    this.body = opts.body;
```
Comment (meeksfred) [inline scripts/portfolio.js:9]: I'm also in the process of adding a new property to my object constructor. I want to add in a small screenshot of each project I have, and plan to update my current template as well.

File: scripts/portfolio.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      Project.loadAll(JSON.parse(localStorage.myData));
+      callback();
+    } else {
+      $.getJSON('/data/portfolioArticles.json', function(myData) {
+        Project.loadAll(myData);
+
+        var toCache = JSON.stringify(myData);
+        localStorage.myData = toCache;
+      });
+      callback();
```
Comment (meeksfred) [inline scripts/portfolio.js:45]: Haven't pushed my updated files yet but they will include the change we discussed here. I believe that this also fixed my category filter as well, which for some reason wasn't pulling data properly.

File: index.html (hunk 1 above)
Comment (jalleng) [inline index.html:56]: Make sure you aren't over-using tools when they aren't necessary.  Handlebars is a great way to add repeated sections to your page.  But if you are only going to have one about section, then there is not a reason to make it into a template.

File: scripts/portfolio.js (hunk 3 above)
Comment (jalleng) [inline scripts/portfolio.js:9]: Screenshots of each project is a good idea.  That's just my opinion though.

**Pull request merged**